test/Libraries/Microsoft.Extensions.Diagnostics.Testing.Tests/Logging/FakeLoggerTests.cs

```diff
@@ -283,4 +281,59 @@ public void Scopes()
         Assert.Equal(42, (int)logger.LatestRecord.Scopes[0]!);
         Assert.Equal("Hello World", (string)logger.LatestRecord.Scopes[1]!);
     }
+
+    [Fact]
+    public void FilterByCustomFilter()
+    {
+        const string NotIgnoredMessage1 = "Not ignored message 1";
+        const string NotIgnoredMessage2 = "Not ignored message 2";
+        const string IgnoredMessage = "Ignored message";
+
+        // Given
+        var options = new FakeLogCollectorOptions
+        {
+            CustomFilters = [
+                r => !r.Message.Equals(IgnoredMessage, StringComparison.Ordinal),
```

Comment:
If you're using Ordinal comparison, then more idiomatic to write as follows:
```suggestion
                r => r.Message != IgnoredMessage,
```